src/hostapi/wdmks/pa_win_wdmks.c

```diff
@@ -4763,6 +4767,10 @@ static PaError OpenStream( struct PaUtilHostApiRepresentation *hostApi,
         {
             stream->render.framesPerBuffer = stream->render.pPin->frameSize;
         }
+        else if(stream->render.framesPerBuffer == 0) {
```

Comment:
If framesPerBuffer==0 AND pPin->frameSize==0 then this clamping logic will get skipped.
It may be safer to not use an else and just do:

    if(stream->render.framesPerBuffer == 0) {